tests/VolleyManagement.UnitTests/Services/UsersService/UserServiceTests.cs

```diff
@@ -67,7 +67,7 @@ public void GetAll_UsersExist_UsersReturned()
             var sut = BuildSUT();
 
             // Act
-            var actual = sut.GetAllUsers();
+            var actual = sut.GetAllUsers().ToList();
```

Comment:
This creates new instance which is not needed. Write your own method to compare collections and put it in TestHelpers class